rocket-app/app/src/main/java/cz/quanti/rocket_app/app/system/MainActivity.kt

```diff
@@ -1,12 +1,9 @@
-package com.example.myapplication
+package cz.quanti.rocket_app.app.system
```

Comment:
take prosim upravit src adresar
src/java -> src/kotlin
(mozna to pujde out of box, ale mozna budes muset pridat sourceSets do build.gradle)

Reply:
done